src/beamlime/applications/base.py

```diff
@@ -12,18 +13,34 @@
     Coroutine,
     Generator,
     List,
+    Mapping,
     Optional,
     Protocol,
+    Tuple,
     runtime_checkable,
 )
 
 from ..logging import BeamlimeLogger
 from ..logging.mixins import LogMixin
 
+EMPTY_ARGS = tuple()
+EMPTY_KWARGS = MappingProxyType(dict())
+
 
 @runtime_checkable
 class MessageProtocol(Protocol):
-    content: Any
+    """Container of arguments and keyword arguments for handlers."""
+
+    args: Tuple
+    kwargs: Mapping
+
+
+@dataclass
+class MessageBase:
+    """Base class for easier implementation of ``MessageProtocol``."""
+
+    args: Tuple = EMPTY_ARGS
+    kwargs: Mapping = EMPTY_KWARGS
```

Comment:
Can't you just use `dataclasses.field(default_factory=tuple)` instead of a global default variable?